api.planx.uk/modules/pay/helpers.ts

```diff
@@ -2,6 +2,17 @@ import { gql } from "graphql-request";
 import airbrake from "../../airbrake.js";
 import { $api } from "../../client/index.js";
 
+/**
+ * Gracefully handle GovPay errors
+ * Docs: https://docs.payments.service.gov.uk/api_reference/#responses
+ */
+export const handleGovPayErrors = (res: unknown) =>
+  JSON.stringify({
+    message:
+      "GovPay responded with an error when attempting to proxy to their API",
```

Comment:
nit: can we simplify this to "GovPay API responded with an error: " please if going to bubble up to frontend error wrapper & be seen by less technical folks? Proxy is detail we/dev team know by default here?